src/dlgConnectionProfiles.cpp

```diff
@@ -716,166 +716,321 @@ QPair<bool, QString> dlgConnectionProfiles::writeProfileData(const QString& prof
+                   "and learn by playing. Challenge non-player characters to gain experience, advance "
+                   "level and maximize your stats. Between battles, sit on the enchanted bench under "
+                   "the Tree to rapidly heal and reduce wait time. Signs in the park present game "
+                   "features such as races, clans and guilds. Read up on teasers about the adventures "
+                   "on the path ahead like dragons, castles and sailing. Join a guild and learn the "
+                   "ways of a Bard, Fighter, Mage, Necromancer, Ninja, Thief, Healer or Priest. Train "
+                   "skills in both craft and combat aligned with your guild. Participate in frequent "
+                   "game-wide events to earn points exchanged for gold, experience or skill training. "
+                   "Heroes and villains alike are invited! Role play is optional and player vs. player "
+                   "combat is allowed in much of the game. StickMUD was born in Finland in June 1991 "
+                   "and is now hosted in Canada. Our diverse community of players and active game "
+                   "engineers are ready to welcome new players like you to one of the best text-based "
+                   "multi-player games ever!");
+    }
+    if (hostUrl == QLatin1String("reinosdeleyenda.es")) {
+/* English translation, provided by Game:
+ *                   "The oldest Spanish free mud with more than 20 years of running history."
+ *                   "\n\n"
+ *                   "Reinos de Leyenda takes place in the ever changing world of Eirea, ravaged by the "
+ *                   "mischiefs of the gods after more than a thousand years of contempt and hideous war "
+ *                   "amongst their zealous mortal pawns."
+ *                   "\n\n"
+ *                   "History is written on a day per day basis, taking into consideration the players' "
+ *                   "choices to decide the irreversible aftermath of this everlasting struggle."
+ *                   "\n\n"
+ *                   "This is a PvP MUD which allows the player to set how high are the stakes: the more "
+ *                   "you risk losing upon death, the more glory to be earned by your heroism. RP, while "
+ *                   "not enforced, is rewarded with non-PvP oriented perks and unique treasure."
+ *                   "\n\n"
+ *                   "A powerful character customization system allows you to choose your deity –or "
+ *                   "fully disregard the gods– and join one of the player-run realms that govern the "
+ *                   "land to explore a breathing world, delve into the secrets of the oceans, shape "
+ *                   "your legacy, craft forgotten marvels for you –or your allies– and fight for faith, "
+ *                   "glory or coin.");
+ */
+        return qsl("El mud Español gratis con más de 20 años de historia."
```

Comment:
Please capitalize MUD like everywhere

```suggestion
        return qsl("El MUD Español gratis con más de 20 años de historia."
```

Reply:
Given that the translation of "Multi User Dungeon" in Spanish appears (according to Google Translate) to be "Dungeon de múltiples usuarios" perhaps we should use "DdMU" - I think we need to get further clarification as to what the correct term ***is*** - and that can follow in a further PR when we get it!

Reply:
It's spelled MUD further down in their text, so I was aiming at some professional alignment here